src/logic/safe/store/actions/createTransaction.ts

```diff
@@ -126,7 +126,7 @@ export const createTransaction =
     const beforeExecutionKey = dispatch(enqueueSnackbar(notificationsQueue.beforeExecution))
 
     let txHash
-    const txArgs: TxArgs & { sender: string } = {
+    const txArgs: TxArgs = {
```

Comment:
Why have you remove `sender` from the arguments?

Reply:
I made it mandatory in `TxArgs` instead.

Reply:
Bizarre. I guess a transaction **always** has a sender!